src/util.js

```diff
@@ -1909,3 +1909,25 @@ export const sync = (dataUpdates={}, contextChildrenUpdates={}, { localOnly, for
     }
   }
 }
+
+/**
+ * Checks if str contains HTML elements
+ * @param {String} str 
+ */
+export const isHTML = (str) => /<\/?[a-z][\s\S]*>/i.test(str);
+
+/**
+ * Replaces certain reserved HTML chars
+ * @param {String} str 
+ */
+export const replaceSpecialHTMLChars = (str) => {
+  const rules = { '(&amp;|&)': 'and' }
```

Comment:
The task does not ask for `&amp;` to be replaced by `and`. Rather, `&` should be allowed and rendered appropriately for the user.